api/base/serializers.py

```diff
@@ -29,6 +29,65 @@ def _url_val(val, obj, serializer, **kwargs):
         return val
 
 
+class HyperLinkedIdentityFieldWithMeta(ser.HyperlinkedIdentityField):
+
+    def __init__(self, view_name=None, **kwargs):
+        assert view_name is not None, 'The `view_name` argument is required.'
+        kwargs['read_only'] = True
+        kwargs['source'] = '*'
+        self.count = kwargs.pop('count', None)
```

Comment:
I don't think we'll change it right now, but something to do in the near future would be to make this more generic, so that we could have an arbitrary function create a metadata object, rather than just giving a count. As it stands, this is more of a `HyperLinkedIdentityFieldWithCount()`.